Formula/fly.rb

```diff
@@ -0,0 +1,14 @@
+class Fly < Formula
+  desc "Concourse fly cli"
+  homepage "https://concourse.ci"
+  url "https://github.com/concourse/concourse/releases/download/v2.3.1/fly_darwin_amd64"
```

Comment:
> it doesn't have a tagged release with the source

The last one is here: https://github.com/concourse/concourse/archive/v2.3.1.tar.gz

> and also uses direnv

`direnv` is available in Homebrew Core. You can add it as a dependency.